src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -97,6 +98,8 @@ public String toSingularFormString() {
             return "team";
         case OWN_TEAM:
             return "team";
+        case CUSTOM:
+            return "custom feedback participant";
```

Comment:
This is used in places where the visibility message is displayed, like the feedbackEdit and feedbackSubmissionEdit pages. This will have to be changed, however I would prefer to do it later, perhaps when I am working on the access control aspect of this feature.